libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/DataType.scala

```diff
@@ -65,7 +65,7 @@ object DataType {
 
   final case class InstallationResult(success: Boolean, code: String, description: String)
 
-  final case class EcuInstallationReport(result: InstallationResult, target: Seq[String], report: Option[Json])
+  final case class EcuInstallationReport(result: InstallationResult, target: Seq[String], rawReport: Option[String])
 
```

Comment:
Why are we relaxing this type, can aktualizr now send reports in formats other than JSON?

Reply:
Yes, it can. In most cases a report is not created by aktualizr itself. It is provided by an external installation manager in a project specific format (ASN.1, for instance).
@jerrytrieu Can we actually define, that a report is always Base64 encoded in a manifest and use Array[Byte] in the field?